uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/components/my-leave-report_test.js

```diff
@@ -0,0 +1,50 @@
+(function (CRM) {
+  define([
+    'common/angular',
+    'common/angularMocks',
+    'leave-absences/shared/config',
+    'leave-absences/my-leave/app'
+  ], function (angular) {
+    'use strict';
+
+    describe('myLeaveReport', function () {
+      var $compile, $log, $rootScope, component, controller;
+
+      beforeEach(module('leave-absences.templates', 'my-leave'));
+      beforeEach(inject(function (_$compile_, _$log_, _$rootScope_) {
+        $compile = _$compile_;
+        $log = _$log_;
+        $rootScope = _$rootScope_;
+
+        spyOn($log, 'debug');
+
+        compileComponent();
+      }));
+
+      it('is initialized', function () {
+        expect($log.debug).toHaveBeenCalled();
+      });
+
+      it('has all the sections collapsed', function () {
+        expect(Object.values(controller.isOpen).every(function (status) {
+          return status === false;
+        })).toBe(true);
+      });
+
+      it('is contains the expected markup', function () {
+        expect(component.find('.chr_leave-report').length).toBe(1);
+      });
+
+      function compileComponent() {
+        var $scope = $rootScope.$new();
```

Comment:
To be honest this is one of my biggest struggles :D I don't think we settled for a pattern yet! I personally prefer using different `var` statements, but we haven't decided anything yet about it